WordPress/UITests/README.md

```diff
@@ -85,17 +85,48 @@ Note that due to the mock server setup, tests cannot be run on physical devices
-* Whether there are repeated flows across tests (defined in `Flows/`).
-* What network requests are made during the test (defined in `API-Mocks/`).
+* Whether to test a user flow (to accomplish a task or goal) or a specific feature (e.g. boundary testing).
+* What screens are being tested (defined as screen objects in [Screens](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITestsFoundation/Screens)).
+* Whether there are repeated flows across tests (defined in [Flows](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITests/Flows)).
+* What network requests are made during the test (defined in [API-Mocks](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/API-Mocks)).
 
-It's preferred to focus UI tests on entire user flows, and group tests with related flows or goals in the same test suite.
+Tests classes are grouped together in [Tests](https://github.com/wordpress-mobile/WordPress-iOS/tree/trunk/WordPress/UITests/Tests)
 
 When you add a new test, you may need to add new screens, methods, and flows. We use page objects and method chaining for clarity in our tests. Wherever possible, use an existing `accessibilityIdentifier` (or add one to the app) instead of a string to select a UI element on the screen. This ensures tests can be run regardless of the device language.
 
+## Naming convention
+
+* When creating new tests, use this format for the name to make it easier to see what the test is doing: `testActionFeature` e.g. `testCreateScheduledPost()`
+* When creating new methods, use this format: `actionObject` e.g. `closePostSettings()`
+* For assert methods, use this format `verifyWhatToVerify` e.g. `verifyPostExists()`
+* Note that there’s a common global method `assertScreenIsLoaded()` that can be used to assert all screens
+
+## Passing hard-coded `Strings` | `Numbers` in tests
+
+There are some cases where we would need to pass hard-coded values in the test, this should happen on the Test level and not on the screen level (unless there’s a really good reason). 
+
+This is so methods are not limited to being used with a fixed value and remain flexible. In the case where those values change we would also be able to update only the test file(s) without making changes elsewhere.
+
 ## Adding or updating network mocks
 
 When you add a test (or when the app changes), the request definitions for WireMock need to be updated in `API-Mocks/`. You can read WireMock’s documentation [here](http://wiremock.org/docs/).
 
-If you are unsure what network requests need to be mocked for a test, an easy way to find out is to run the app through [Charles Proxy](https://www.charlesproxy.com/) and observe the required requests.
+If you are unsure what network requests need to be mocked for a test, an easy way to find out is to run the app through [Proxyman](https://proxyman.io/) or [Charles Proxy](https://www.charlesproxy.com/) and observe the required requests.
```

Comment:
```suggestion
If you are unsure what network requests need to be mocked for a test, a way to find this out is to run the app through [Proxyman](https://proxyman.io/) or [Charles Proxy](https://www.charlesproxy.com/) and observe the required requests.
```

(No, just joking 🙂)

Reply:
that is a pretty valid suggestion though 😆